src-docs/src/views/link/link.js

```diff
@@ -9,14 +9,15 @@ export default () => (
       <EuiLink href="http://www.elastic.co" target="_blank">
         Elastic website
       </EuiLink>{' '}
-      in a new tab.
+      in a new tab. Setting <EuiCode>target=&ldquo;_blank&rdquo;</EuiCode> also
+      defaults to <EuiCode>external={true}</EuiCode>.
     </p>
     <p>
       This{' '}
-      <EuiLink href="http://www.elastic.co" external target="_blank">
+      <EuiLink href="http://www.elastic.co" external={false} target="_blank">
```

Comment:
We need one that shows `external=true`. So lets just remove the target.
```suggestion
      <EuiLink href="http://www.elastic.co" external>
```

Reply:
Do we need another then showing the interplay between `external` and `target`?

Reply:
I wouldn't just because we probably don't really want them to. 😉